lib/rules/no-sync.js

```diff
@@ -57,8 +122,31 @@ module.exports = {
              * @returns {void}
              */
             [selector.join(",")](node) {
-                if (ignores.includes(node.name)) {
-                    return
+                const parserServices = getParserServices(context)
+
+                for (const ignore of ignores) {
+                    if (typeof ignore === "string") {
+                        if (ignore === node.name) {
+                            return
+                        }
+
+                        continue
+                    }
+
+                    if (parserServices !== null) {
```

Comment:
I am torn here, as I think that if someone misses the two mentions of ts only in the documentation it wont be obvious that a `{ from: "file" }` rule in their config was skipped.

Part of me wants to throw an error or make it more obvious, what do you think?

Reply:
Yeah, throwing an error, or at least logging a warning would be a good idea.